frontend/src/views/offertable/constants.js

```diff
@@ -0,0 +1 @@
+export const SEND_SELECTED_ROWS = "SEND_SELECTED_ROWS";
```

Comment:
Why this name?

Reply:
Because the table just passes the selected rows out, and for other components that need selected rows, they are just reading the rows from the store. So this is more like an action defined for the table I think. Hmmm, but now I think `SET_SELECTED_ROWS` might be better.